docs/benchmarks/_index.md

```diff
@@ -28,6 +28,8 @@ How to setup the environment for the Kafka Performance Test.
 
 ## EKS
 
+{{< include-headless "warning-ebs-csi-driver.md" "sdm/koperator" >}}
```

Comment:
@fekete-robert can you help double check if this warning would look as expected?